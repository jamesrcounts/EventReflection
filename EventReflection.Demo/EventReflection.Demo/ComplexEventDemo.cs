using Microsoft.VisualStudio.TestTools.UnitTesting;

namespace EventReflection.Demo
{
    [TestClass]
    public class ComplexEventDemo
    {
        [TestMethod]
        public void PocoExtensionTest()
        {
            var target = new PocoExtension();
            target.ProcessStarted += Domain.HandleProcessStarted;
            target.PropertyChanged += Domain.HandlePropertyChanged;
            EventUtility.VerifyEventCallbacks(target);
        }
    }
}