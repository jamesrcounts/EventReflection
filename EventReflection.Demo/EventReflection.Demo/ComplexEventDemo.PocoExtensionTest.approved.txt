Event callbacks for PocoExtension

PropertyChanged
	[0] Void HandlePropertyChanged(System.Object, System.ComponentModel.PropertyChangedEventArgs)

ProcessStarted
	[0] Void HandleProcessStarted(System.Object, System.EventArgs)

